cookiecutter.json

```diff
@@ -13,14 +13,15 @@
     "docker_base": [
         "alpine:3.7",
         "alpine:3.8",
+        "centos:7",
+        "custom:special-image",
```

Comment:
I would be a bit more thatn "special-image" here so we do not end up having "special-image1", "special-image2" :-)

Reply:
well there should be only one custom... as basically the aim is to allow to use whatever base in the dockerfile... so if we end up having a special image that becomes very trendy we would add it to the list of available images.

Reply:
awesome!!